### ISSUE TITLE
L2 regularization loss is scaled twice.

### ISSUE THREAD

**Issue by willthefrog** [ncullen93/torchsample#42]
see regularizers.py#L51
```python
value = th.sum(th.pow(w,2)) * self.scale
loss = self.scale * value
```

**Comment by benwu232** [comment]
Just found the same issue!

**Comment by ncullen93** [comment]
fixed in new release. thanks :)
